File: aula03/app/src/main/java/com/example/eric_aula03/MainActivity.java
package com.example.eric_aula03;


import android.os.Bundle;
import android.widget.Button;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Livro livro1, livro2, livro3;

        livro1 = new Livro("Desenvolvendo seu primeiro APlicativo", "Ericson", 2024);
        livro2 = new Livro("Desenvolvendo seu primeiro APlicativo", "Ericvon", 2024);
        livro3 = new Livro("Desenvolvendo seu primeiro APlicativo", "Ericton", 2024);

        TextView textInfo = findViewById(R.id.textInfo);


        Button button1 = findViewById(R.id.button1);
        Button button2 = findViewById(R.id.button2);
        Button button3 = findViewById(R.id.button3);

        button1.setOnClickListener(v -> textInfo.setText(livro1.getinfo()));
        button2.setOnClickListener(v -> textInfo.setText(livro2.getinfo()));
        button3.setOnClickListener(v -> textInfo.setText(livro3.getinfo()));
    }

}

File: Aula4/app/src/main/java/com/example/aula/MainActivity.kt
package com.example.aula

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private var editNome: EditText? = null
    private var btnEnviar: Button? = null

    @Override
    protected fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main) // Layout da primeira tela

        editNome = findViewById(R.id.editNome)
        btnEnviar = findViewById(R.id.btnEnviar)

        btnEnviar.setOnClickListener(object : OnClickListener() {
            @Override
            fun onClick(v: View?) {
                val nomeDigitado: String = editNome.getText().toString()

                val irParaSegundaTela: Intent = Intent(
                    this@MainActivity,
                    SegundaActivity::class.java
                )
                irParaSegundaTela.putExtra("nomeUsuario", nomeDigitado)
                startActivity(irParaSegundaTela)
            }
        })
    }
}